pubspec.yaml

```diff
@@ -12,6 +12,9 @@ dependencies:
   mason_logger: ^0.2.2
   meta: ^1.8.0
   path: ^1.8.2
+screenshots:
+  - description: 'This screenshot shows the completion of a dart CLI.'
+    path: doc/screen.png
```

Comment:
should this be an absolute url? I have a feeling this may not work once we publish to pub.dev